packages/altair-app/src/app/modules/altair/config.ts

```diff
@@ -145,6 +164,7 @@ export class AltairConfig {
     this.initialData.postRequestScript = initialPostRequestScript;
     this.initialData.instanceStorageNamespace = (window as any).__ALTAIR_INSTANCE_STORAGE_NAMESPACE__ || instanceStorageNamespace || 'altair_';
     this.initialData.settings = initialSettings || undefined;
+    this.initialData.initialSubscriptionsProvider = initialSubscriptionsProvider || "websocket";
```

Comment:
You should use the subscription provider constant instead of hardcoding `websocket` here.